### DISCUSSION TITLE
Help needed - y21ga, y29ga? unbrick

### DISCUSSION THREAD

**Discussion by Maxxmisiek** [roleoroleo/yi-hack-Allwinner-v2#411] [Q&A]
Hello and welcome.

Recently, I got my hands on the yi home (bfus) webcam shining yellow all the time

After disassembling, I found the marking y29ga

Any means of reviving it had not worked.
After connecting the serial cable, I got info that it was the y21ga version -
Unfortunately, unbrick and unhack didn't work and still glows nice yellow.

But after loading the bootloader and issuing the RESET command, the camera magically comes alive and works - you can pair, login etc.
Unfortunately, after a power failure, the process stops in the same place.

Software 9.0.19 ..... - unfortunately there is nothing newer by the application

Maybe someone has an idea or information on how to throw all soft into this "great machinery"

I do not have any backup to it because the equipment was accidentally found at a friend's. I will also not change her past - was there anything done with her.

Some code 
`77]HELLO! BOOT0 is starting Nov 14 2019 20:01:31!
[82]BOOT0 commit : 81c18ee
[85]board init start
[87]set pll start
[89]set pll end
[91][pmu]: bus read error
[93]board init ok
[95]chip id check OK
[97]DRAM BOOT DRIVE INFO: V0.41
[100]DRAM CLK = 528 MHz
[102]DRAM Type = 2 (2:DDR2,3:DDR3)
[106]DRAMC read ODT  off.
[108]DRAM ODT[190]Entry_name        = optee
[193]Entry_name        = u-boot
[197]Entry_name        = dtb
[201]Jump to secend Boot.
MESSAGE: [0x0] TEE-CORE: OP-TEE version: sun8iw19p1_v0.6.0-12-g97f2688 #1 2019年 11月 08日 星期五 08:26:51 UTC arm
ERROR:   [0x0] TEE-CORE:platform_standby_fdt_parse:126: no pmu node
ERROR:   [0x0] TEE-CORE:sunxi_twi_parse_from_dt:84: no pmu node

U-Boot 2018.05-00007-g8b9203a (Jul 14 2020 - 14:49:11 +0800) Allwinner Technology

[00.246]DRAM:  64 MiB
[00.248]Relocation Offset is: f9f94000
[00.260]secure enable bit: 0
[00.262]CPU=816 MHz,PLL6=600 Mhz,AHB=200 Mhz, APB1=100Mhz  MBus=132Mhz
[00.268]gic: sec monitor mode
[00.271]flash init start
[00.273]workmode = 0,storage type = 3
SF: Detected w25q64cv with page size 256 Bytes,OK
[00.354]try sprite_led_gpio config
xxxxxxxxxxxxxxlijun uboot xiaoyiledinit
[00.361]sprite_led_gpio start
[00.365](weak)update dtb dram start
[00.369]update dtb dram  end
[00.371]update dts
root_partition is rootfs
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
set root to /dev/mtdblock2
[00.434]update part info
[00.436]update bootcmd
mmc driver ver uboot2018:2019-8-15 16:34:00
get mem for descripter OK !
[00.449]get sdc_wipe fail.
[00.451]get sdc0 sdc_erase fail.
[00.454]get sdc0 sdc_boot fail.
[00.457]get sdc0 sdc_odly_50M fail.
[00.460]get sdc0 sdc_sdly_50M fail.
[00.463]get▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[00.694]init mmc 0 clock and io
devnum 0, prv 43fbf410, bdesc 42b7476c
SUNXI SD/MMC: 0
[00.701]==========OOOOOOOOO▒▒ɭ mode: 0 0, sample_mode:0
[00.706]=============== start mmc_init_boot...
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒card_caps:0x3000000a
host_cX▒:0x3000003f
[00.762]unable to read ssr
[▒0.765]unable to read s.5
                          [00.768]the mode SD Legacy (freq : 25 MHz)
** Unrecognized file▒▒ѕ▒ type **
c▒▒▒▒e▒▒▒ [/one_y21ga.bi▒] exsit
start: 0x2e0, len: 0x1
start: 0x2e1, len:(0x1
start: 0x2e2, len: 0x2
parname:boot start_offset:0x20 part_size:1900544
start: 0x2e0, len: 0x1
start: 0x2e1,b▒▒: 0x1
start: 0x2e2, len: 0x2
part_name:rootfs start_offset:0xea0 part_size:1179648
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
part_name:home start_offset:0x17a0 part_size:3407872
tart: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2m2, len: 0xFHpart_name;backup start_offset:0x31a0 part_size:1245184
start: 0x2e0, len: 0x1
st▒rt: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
part_name:env start_offset:0x3b20 pcrt_size:131072
Nothing updated, start normal boot
Hit any key to stop autoboot:  0
=> 
`

After RESET via serial console

`Hit any key to stop autoboot:  0
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
[01.956]partinfo: name boot, start 0x20, size 0xe80
start: 0x300, len: 0x40
start: 0x340, len: 0xe11
[02.405]android.hardware = sun8iw19p1
## Booting kernel from Legacy Image at 45000000 ...
   Image Name:   ARM OpenWrt Linux-4.9.118
   Image Type:   ARM Linux Kernel Image (uncompressed)
   Data Size:    1876232 Bytes = 1.8 MiB
   Load Address: 40008000
   Entry Point:  40008000
[02.472]Starting kernel ...

** 2 printk messages dropped ** [    0.000000] ion_mem_reserve: err format for ion reserve list!
[    0.099506] ion_heap_create: Invalid heap type 4
[    0.106577] sunxi_i2c_probe()2209 - [i2c1] warning: failed to get regulator id
[    0.129516] uart uart0: get regulator failed
[    0.160093] sunxi-wlan wlan: request pincrtl handle for device [wlan] failed
[    0.168364] sunxi-wlan wlan: get gpio chip_en failed
[    0.189885] mtd: partition "UDISK" is out of reach -- disabled
[    0.284731] VE: get debugfs_mpp_root is NULL, please check mpp
[    0.284731]
[    0.558155] mmc:failed to get gpios
[    0.606504] sunxi-mmc sdc1: smc 1 p1 err, cmd 52, RTO !!
[    0.613379] sunxi-mmc sdc1: smc 1 p1 err, cmd 52, RTO !!
[    0.624519] sunxi-mmc sdc1: smc 1 p1 err, cmd 8, RTO !!
[    0.631282] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.638047] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.644812] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.651548] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.658368] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.665266] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.672178] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.678771] --->sunxi_daudio_dev_probe<--- strat.
[    0.684133] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.690609] sunxi-mmc sdc1: smc 1 p1 err, cmd 1, RTO !!
[    0.700002] --->sunxi_daudio_dev_probe<--- end.
starting pid 532, tty '': '/etc/init.d/rcS boot'
[RCS]: /etc/init.d/S01udev
Populating /dev using udev: [    1.265041] udevd[537]: could not create /run/udev: No such file or directory
done
[RCS]: /etc/init.d/S02app
[RCS]: /etc/init.d/S20urandom
--------------------------home app init.sh--------------------------
NETWORK_IFACE=wlan0
sh: write error: Invalid argument
read-only file system detected...done
starting pid 550, tty '/dev/console': '-/bin/sh'

BusyBox v1.27.2 () built-in shell (ash)

vm.dirty_background_ratio = 2
vm.dirty_expire_centisecs = 500
vm.dirty_ratio = 2
vm.dirty_writeback_centisecs = 100

            __    __   _
            \ \  / /  | |
             \ \/ /   | |
              \  /    | |
              / /     | |
             /_/      |_|
 ------------------------------------
 Welcome to XiaoYi Linux.
 ------------------------------------

root@(none):/# fs.mqueue.msg_max = 256

checkdisk.c(get_tfcard_size_v2-105)[00:00:32.456]:TF card total size : 14804 MB

checkdisk.c(get_fs_type-251)[00:00:32.458]:FS TYPE : VFAT

checkdisk.c(get_cluster_size-313)[00:00:32.502]:cluster_size = 16384 Bytes

checkdisk.c(check_partion_vfat-152)[00:00:32.502]:TF Card  checking ....

checkdisk.c(do_fat_check-117)[00:00:32.502]:Can fat check

checkdisk.c(do_fat_check-121)[00:00:32.949]:retBuf(fsck.fat 3.0.26 (2014-03-07)
step 0 32
step 1 32
step 2 32
step 3 32
step 4 32
step 5 32
step 6 32
/dev/mmcblk0p1: 101 files, 13657/946550 clusters
)

checkdisk.c(do_fat_check-140)[00:00:32.952]:Mount sd ok.

checkdisk.c(check_partion_vfat-156)[00:00:32.952]:TF Card check finished.

checkdisk, main() ret_status(10000)
checkdisk, main() ret_status(10000),g_sysconf_ptr->tf_status.stat(10000)
---update file(home_y21gam) Not exist---
--------------------------insmod sensor--------------------------
[    3.820867] sunxi-mmc sdc1: smc 1 p1 err, cmd 8, RTO !!
[    5.799472] sunxi-gmac 5020000.eth eth0: Initialize hardware error
============================================= home low_half_init.sh... =========================================
============================================= begin to start app... =========================================

dispatch.c(get_productinfo-1430)[00:00:35.595]:get device id info: did[BFUSY44XK7I5LX200807], key[73bpJxz28ZtYtlqf], key_mi[FFFFFFFFFFFFFFFF], p2pid[TNPUSAL-119339-CMESY], did_mi[00000000]

hw12_ab_cd_ratio[0],hw23_26_x_step:[0],hw27_30_y_step:[0]

dispatch.c(get_config-2075)[00:00:38.042]:got sn(

dispatch.c(get_config-2076)[00:00:38.042]:got pwd(

`

**Comment by roleoroleo** [comment]
Bootloader is properly working.
So the unbrick procedure must work:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/tree/master/unbrick

If it doesn't work , try to change the sd card or change the file to load.

**Reply by Maxxmisiek** [reply]
Hello.  Unfortunately, nothing worked.  The unbrick or unhack procedure does not start when placed on the sd card.  Only after calling the Reset command by the com consoles, the procedures start, but after the next power-up the camera does not start again.  It seems to me that the problem is precisely the bootloader or some incorrect env entries.  Maybe the mythical one_y21 file would help?

**Reply by roleoroleo** [reply]
If the log is not changed from the message above, the bootloader is ok.
Yes, the file one_y21ga could help.
But backup_y21ga should work.
Did you try to create the unbrick file with "hacked" option?

**Reply by Maxxmisiek** [reply]
> If the log is not changed from the message above, the bootloader is ok. Yes, the file one_y21ga could help. But backup_y21ga should work. Did you try to create the unbrick file with "hacked" option?

Hi Roleo

For a long time I did not try to revive the camera.
But I try again.
Unfortunately, I don't have a backup to try the unhack procedure.
Can you send me a complete backup of the y21ga camera?

**Comment by Maxxmisiek** [comment]
Hi Rolero 

For a long time I did not try to revive the camera.

But I try again.

Unfortunately, I don't have a backup to try the unhack procedure.

Can you send me a complete backup of the y21ga camera?

**Comment by roleoroleo** [comment]
Try this:
[backup_y21ga.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/10713905/backup_y21ga.gz)

**Comment by Maxxmisiek** [comment]
Unfortunately . I guess I'm out of luck. It still glows orange. I am wondering that after stopping and issuing the RESET command by the console, it starts correctly. Maybe something with the BOOT or ENV partition? Unfortunately I don't have them :-(

**Comment by alienatedsec** [comment]
Hi @roleoroleo 

I will continue the same subject with my `y29ga` - `IFUS` model that got stuck for some reason. I do have backups of `mtdblock[2,4,6,7].bin`; however, failing to unbrick or recover. Please can you have a look at those logs?

```
[38]HELLO! BOOT0 is starting Nov 14 2019 20:01:31!
[43]BOOT0 commit : 81c18ee
[46]board init start
[48]set pll start
[50]set pll end
[52][pmu]: bus read error
[54]board init ok
[56]chip id check OK
[58]DRAM BOOT DRIVE INFO: V0.41
[61]DRAM CLK = 528 MHz
[63]DRAM Type = 2 (2:DDR2,3:DDR3)
[67]DRAMC read ODT  off.
[69]DRAM ODT off.
[71]DRAM SIZE =64 M
[78]DRAM simple test OK.
[80]rtc standby flag is 0x0, super standby flag is 0x0
[86]dram size =64
[88]spinor id is: c8 40 17, read cmd: 03
[92]Succeed in reading toc file head.
[96]The size of toc is 4c000.
[149]Entry_name        = optee
[152]Entry_name        = u-boot
[156]Entry_name        = dtb
[159]Jump to secend Boot.
MESSAGE: [0x0] TEE-CORE: OP-TEE version: sun8iw19p1_v0.6.0-12-g97f2688 #1 2019年              11月 08日 星期五 08:26:51 UTC arm
ERROR:   [0x0] TEE-CORE:platform_standby_fdt_parse:126: no pmu node
ERROR:   [0x0] TEE-CORE:sunxi_twi_parse_from_dt:84: no pmu node


U-Boot 2018.05-00007-g8b9203a (Jul 14 2020 - 14:49:11 +0800) Allwinner Technolog             y

[00.205]DRAM:  64 MiB
[00.207]Relocation Offset is: f9f94000
[00.219]secure enable bit: 0
[00.221]CPU=816 MHz,PLL6=600 Mhz,AHB=200 Mhz, APB1=100Mhz  MBus=132Mhz
[00.228]gic: sec monitor mode
[00.230]flash init start
[00.232]workmode = 0,storage type = 3
SF: Detected gd25q64 with page size 256 Bytes, erase size 4 KiB, total 8 MiB
[00.251]sunxi flash init ok
[00.254]Loading Environment from SUNXI_FLASH... start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x3e00, len: 0x100
OK
[00.315]try sprite_led_gpio config
xxxxxxxxxxxxxxlijun uboot xiaoyiledinit
[00.322]sprite_led_gpio start
[00.325](weak)update dtb dram start
[00.330]update dtb dram  end
[00.332]update dts
root_partition is rootfs
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
set root to /dev/mtdblock2
[00.395]update part info
[00.397]update bootcmd
mmc driver ver uboot2018:2019-8-15 16:34:00
get mem for descripter OK !
[00.410]get sdc_wipe fail.
[00.412]get sdc0 sdc_erase fail.
[00.415]get sdc0 sdc_boot fail.
[00.418]get sdc0 sdc_odly_50M fail.
[00.421]get sdc0 sdc_sdly_50M fail.
[00.424]get sdc0 sdc_odly_50M_ddr fail.
[00.428]get sdc0 sdc_sdly_50M_ddr fail.
[00.431]get sdc0 sdc_freq fail.
[00.434]get sdc0 sdc_b0p fail.
[00.437]get card0_boot_para:sdc_ex_dly_used fail
[00.442]get card-pwr-gpios handler:1119307304
[00.446]get card0_boot_para:time_pwroff:200ms
[00.451]Using default timing para
[00.654]init mmc 0 clock and io
devnum 0, prv 43fbd410, bdesc 42b7476c
SUNXI SD/MMC: 0
[00.662]==================== work mode: 0 0, sample_mode:0
[00.667]=============== start mmc_init_boot...
[00.686]Card did not respond to voltage select!
[00.690]mmc_init: -95, time 28
[00.692]mmc_init: mmc init fail, err -95
card0 init failed
No sd-card insert,will normal boot
Hit any key to stop autoboot:  0
start: 0x2e0, len: 0x1
start: 0x2e1, len: 0x1
start: 0x2e2, len: 0x2
[01.715]partinfo: name boot, start 0x20, size 0xe80
start: 0x300, len: 0x40
start: 0x340, len: 0xe11
[02.164]android.hardware = sun8iw19p1
## Booting kernel from Legacy Image at 45000000 ...
   Image Name:   ARM OpenWrt Linux-4.9.118
   Image Type:   ARM Linux Kernel Image (uncompressed)
   Data Size:    1876232 Bytes = 1.8 MiB
   Load Address: 40008000
   Entry Point:  40008000
[02.231]Starting kernel ...

** 2 printk messages dropped ** [    0.000000] ion_mem_reserve: err format for i             on reserve list!
[    0.097889] ion_heap_create: Invalid heap type 4
[    0.104063] sunxi_i2c_probe()2209 - [i2c1] warning: failed to get regulator i             d
[    0.127890] uart uart0: get regulator failed
[    0.158380] sunxi-wlan wlan: request pincrtl handle for device [wlan] failed
[    0.166662] sunxi-wlan wlan: get gpio chip_en failed
[    0.188442] mtd: partition "UDISK" is out of reach -- disabled
[    0.282851] VE: get debugfs_mpp_root is NULL, please check mpp
[    0.282851]
[    0.555152] mmc:failed to get gpios
[    0.598733] sunxi-mmc sdc1: smc 1 p1 err, cmd 52, RTO !!
[    0.605581] sunxi-mmc sdc1: smc 1 p1 err, cmd 52, RTO !!
[    0.616034] sunxi-mmc sdc1: smc 1 p1 err, cmd 8, RTO !!
[    0.622802] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.629577] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.636348] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.643126] sunxi-mmc sdc1: smc 1 p1 err, cmd 5, RTO !!
[    0.649907] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.656761] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.663663] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.670558] sunxi-mmc sdc1: smc 1 p1 err, cmd 55, RTO !!
[    0.676942] --->sunxi_daudio_dev_probe<--- strat.
[    0.682280] sunxi-mmc sdc1: smc 1 p1 err, cmd 1, RTO !!
[    0.689591] --->sunxi_daudio_dev_probe<--- end.
[    0.788815] Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(31,2)
[    0.798222] ---[ end Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(31,2)

```

**Reply by alienatedsec** [reply]
Managed to recover `rootfs` from another copy found in #329 

```
starting pid 549, tty '/dev/console': '-/bin/sh'


BusyBox v1.27.2 () built-in shell (ash)


            __    __   _
            \ \  / /  | |
             \ \/ /   | |
              \  /    | |
              / /     | |
             /_/      |_|
 ------------------------------------
 Welcome to XiaoYi Linux.
 ------------------------------------

root@(none):/#
```

I will follow with the same for the `home` partition.

**Reply by alienatedsec** [reply]
Success - working again :)

```

p2p_tnp.c(p2p_send_stop_viewing-2115)[10:18:22.790]:p2p_send_stop_viewing send_msg ok!


p2p_tnp.c(tnp_worker-6491)[10:18:22.790]:---0, frame_trans exit!!


v429_mpp.c(human_motion_process-2299)[10:18:22.913]:motion stop. send_motion_stop_msg


rmm.c(send_motion_stop_msg-1852)[10:18:22.913]:msg snd success

```

**Reply by roleoroleo** [reply]
Sorry I wasn't able to respond sooner.
Glad you solved.

**Comment by endrrs** [comment]
Hi roleoroleo,

I have two of the y291ga cameras that is half bricked. I tried the software unbricking, but no change.
After reboot and short shutter sound just blinking orange light.
Today I did the soldering and Putty log, but I failed to stop it at boot, it was not responding.

Can you have a look at the log and advise what the problem is and possible remedy. Thanks.

```
[143]HELLO! BOOT0 is starting!
[146]BOOT0 commit : 7334c37ba
[149]board init start
[151]set pll start
[154]set pll end
[155][pmu]: bus read error
[158][pmu]: bus read error
[160]board init ok
[162]chip id check OK
[164]DRAM BOOT DRIVE INFO: V0.41
[168]DRAM CLK = 528 MHz
[170]DRAM Type = 2 (2:DDR2,3:DDR3)
[173]DRAMC read ODT  off.
[176]DRAM ODT off.
[178]DRAM SIZE =64 M
[185]DRAM simple test OK.
[188]rtc standby flag is 0x0, super standby flag is 0x0
[193]dram size =64
[195]key press : 
[198]spinor id is: 5e 40 17, read cmd: 0b
[202]Succeed in reading toc file head.
[205]The size of toc is 4c000.
[290]Entry_name        = optee
[293]Entry_name        = u-boot
[297]Entry_name        = dtb
[300]Jump to second Boot.
MESSAGE: [0x0] TEE-CORE: OP-TEE version: sun8iw19p1_v0.6.0-27-g7f6d3375 #1 Thu Dec 10 03:28:16 UTC 2020 arm
ERROR:   [0x0] TEE-CORE:platform_standby_fdt_parse:126: no pmu node
ERROR:   [0x0] TEE-CORE:sunxi_twi_parse_from_dt:127: no pmu node


U-Boot 2018.05 (Nov 29 2021 - 13:29:53 +0800) Allwinner Technology

[00.344]DRAM:  64 MiB
[00.346]Relocation Offset is: f9f94000
[00.358]secure enable bit: 0
[00.360]CPU=816 MHz,PLL6=600 Mhz,AHB=200 Mhz, APB1=100Mhz  MBus=132Mhz
[00.366]gic: sec monitor mode
[00.369]flash init start
[00.371]workmode = 0,storage type = 3
SF: Detected ZB25VQ64 with page size 256 Bytes, erase size 64 KiB, total 8 MiB
[00.388]sunxi flash init ok
[00.391]Loading Environment from SUNXI_FLASH... OK
[00.420]try sprite_led_gpio config
xxxxxxxxxxxxxxlijun uboot xiaoyiledinit

[00.428]sprite_led_gpio start
[00.431](weak)update dtb dram start
[00.435]update dtb dram  end
[00.438]update dts
root_partition is rootfs
set root to /dev/mtdblock2
[00.447]update part info
[00.450]update bootcmd
mmc driver ver uboot2018:2019-8-15 16:34:00
get mem for descripter OK !
[00.462]get sdc_wipe fail.
[00.464]get sdc0 sdc_erase fail.
[00.467]get sdc0 sdc_boot fail.
[00.470]get sdc0 sdc_odly_50M fail.
[00.473]get sdc0 sdc_sdly_50M fail.
[00.477]get sdc0 sdc_odly_50M_ddr fail.
[00.480]get sdc0 sdc_sdly_50M_ddr fail.
[00.484]get sdc0 sdc_freq fail.
[00.487]get sdc0 sdc_b0p fail.
[00.489]get card0_boot_para:sdc_ex_dly_used fail
[00.495]get card-pwr-gpios handler:1119307360
[00.498]get card0_boot_para:time_pwroff:200ms
[00.504]Using default timing para
card_caps:0x3000000a
host_caps:0x3000003f
[00.838]unable to read ssr
[00.840]unable to read ssr
[00.843]the mode SD Legacy (freq : 25 MHz)
check_file [/one_y291ga.bin] exsit 
check_file [/one_y291ga.bin] exsit 
part_name:boot start_offset:0x20 part_size:1900544 
part_name:rootfs start_offset:0xea0 part_size:1179648 
part_name:home start_offset:0x17a0 part_size:3407872 
part_name:backup start_offset:0x31a0 part_size:1245184 
part_name:env start_offset:0x3b20 part_size:131072 
Nothing updated, start normal boot
debug_tty, print(<NULL>) 
card0 has inited
debug_tty, can't find debug_tty or Factory in sd !!! 
Hit ctrl + q  to stop autoboot:  0 
[00.944]partinfo: name boot, start 0x20, size 0xe80
[01.389]android.h!rdware = sun8iw19p1
## Booting kernel from Legacy Image at 45000000 ...
   Image Name:   ARM OpenWrt Linux-4.9.118
   Image Type:   ARM Linux Kernel Image (uncompressed)
   Data Size:    1877976 Bytes = 1.8 MiB
   Load Address: 40008000
   Entry Point:  40008000
[01.456]Starting kernel ...
```

**Comment by roleoroleo** [comment]
The log is ok, nothing strange.
The lines:
```
check_file [/one_y291ga.bin] exsit 
check_file [/one_y291ga.bin] exsit 
part_name:boot start_offset:0x20 part_size:1900544 
part_name:rootfs start_offset:0xea0 part_size:1179648 
part_name:home start_offset:0x17a0 part_size:3407872 
part_name:backup start_offset:0x31a0 part_size:1245184 
part_name:env start_offset:0x3b20 part_size:131072 
Nothing updated, start normal boot
```
mean that you copied the whole image into the sd card (one_y291ga.bin) but all partitions are updated. The bootloader checks the hash of each single partition.

If after "Starting kernel" the cam freezes, probably you need to  rewrite the kernel image. But I never seen a corruption in kernel partition, it's very strange.
Before overwriting the kernel, please try again to upload mtdblock4 created using the command `./build.sh hacked`
